web/react/components/channel_header.jsx

```diff
@@ -55,7 +55,7 @@ export default class ChannelHeader extends React.Component {
         if (!Utils.areStatesEqual(newState, this.state)) {
             this.setState(newState);
         }
-        $('.channel-header__info .description').popover({placement: 'bottom', trigger: 'hover click', html: true, delay: {show: 500, hide: 500}});
```

Comment:
Removed the click event for the description popover since I don't think it's required.